sdk/nodejs/__tests__/eventQueue.test.ts

```diff
@@ -5,13 +5,14 @@ import { AxiosResponse } from 'axios'
 import { EventQueue, EventQueueOptions } from '../src/eventQueue'
 import { EventTypes } from '../src/eventQueue'
 import { publishEvents } from '../src/request'
-import { BucketedUserConfig, DVCLogger, PublicProject } from '@devcycle/types'
+import { BucketedUserConfig, DVCLogger, DVCReporter, FlushResults, PublicProject } from '@devcycle/types'
```

Comment:
based on the warning

```suggestion
import { BucketedUserConfig, DVCLogger, DVCReporter, PublicProject } from '@devcycle/types'
```